### DISCUSSION TITLE
LoRaWAN on Brazil with Zephyr and neutral network

### DISCUSSION THREAD

**Discussion by GABS9718** [zephyrproject-rtos/zephyr#36215] [Q&A]
I have been trying to use LoRaWAN + Zephyr + neutral network (provided by American Tower) in Brazil, but I'm having some problems as I can not make a join in the network. What I keep receive can be seen in the image below

![image](https://user-images.githubusercontent.com/60732727/121829828-ff497980-cc99-11eb-90e6-09af147ab159.png)

I'm currently using the following prj.conf and the samples/lorawan/class_a example with the correct keys and OTAA mode. My hardware is a nrf52840 + sx1276 module.
```
#GENERAL SETTINGS
CONFIG_CPLUSPLUS=y
CONFIG_REBOOT=y
CONFIG_NEWLIB_LIBC=y

#LOG
CONFIG_LOG=y
CONFIG_USE_SEGGER_RTT=y
CONFIG_RTT_CONSOLE=y
CONFIG_UART_CONSOLE=n
CONFIG_PRINTK=y

#LORAWAN
CONFIG_SPI=y
CONFIG_SPI_NRFX=y
CONFIG_SPI_1=y
CONFIG_GPIO=y
CONFIG_LORA=y
CONFIG_LORA_SX12XX=y
CONFIG_LORAWAN=y
CONFIG_LORAMAC_REGION_AU915=y
CONFIG_SHELL=y
```
Inside RegionAU915.h there are the physical settings for the LoRaWAN protocol. My tip is that the project is not currently working on with Zephyr as these settings are not the same as the network is expecting. I should have something like this https://ns.docs.everynet.io/channel_plans/LA915A.html on the RegionAU915.h file. 

Based on this documentation that I posted, can someone try to help me on which can be the path to follow to debug and understand why zephyr is not working with this neutral network? Basically, what I need to understand is if the RegionAU915.h file is with the settings that I need (https://ns.docs.everynet.io/channel_plans/LA915A.html). 

 @Mani-Sadhasivam

**Comment by GABS9718** [comment]
@nashif

**Comment by tcpipchip** [comment]
i am having the same problem!
I am using the STM32L072+SX1276
Always getting RX 2 TIMEOUT...
and in the LORAWAN SIDE a report that the MIC is incorrect
I am investing if DIO5 is really being used :(
https://lom204-cli-wisol.blogspot.com/2022/11/o-objetivo-deste-blog-e-demonstrar-como.html
